lexnlp/extract/en/dates.py

```
@@ -68,280 +70,295 @@ def get_month_by_name():
-) -> bool:
-    """
-    Checks that when we transformed "possible date" into date, we found
-    place for each "token" from the initial phrase
-    :param date:
-    :param date_string: "13.2 may"
-    :param date_props: {'time': [], 'hours': [] ... 'digits': ['13', '2'] ...}
-    :return: True if date is OK
-    """
+    date_finder = DateFinder(base_date=base_date)
+    for extra_token in date_finder.EXTRA_TOKENS_PATTERN.split('|'):
+        if extra_token != 't':
```

Ruff (0.15.4)

[error] Possible hardcoded password assigned to: "extra_token"

(S105)